captcha.html

```diff
@@ -276,6 +276,13 @@
         line-height: 2.5rem
       }
     }
+
+    div.g-recaptcha {
+      margin: 0 auto;
+      width: 304px;
+      margin-top: 20px;
+      margin-bottom: 20px;
```

Comment:
Your margins are very weird, maybe you can use `margin: 20px auto 20px`

Additionnaly, not very sure about set a width to this div.

You can play with `display: flex; justify-items: center;`